public/index.html

```diff
@@ -2,11 +2,11 @@
 <html>
   <head>
 		<title>TROMPA Digital Score Edition</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="%PUBLIC_URL%/style.css">
+    <script src="%PUBLIC_URL%/verovio-toolkit.js"></script>
```

Comment:
this change is a bit delicate... because meld-clients-core uses the `verovio` global (and needs it to be there as soon as the js is loaded), this has to be loaded before the bundle, which is automatically added after the `<body>` tag. 
I tried to add verovio from npm, but it caused node to run out of memory and crash. Because we're using react-scripts, there's no way to add verovio to the excludes of the webpack configuration. We need to look and see if there's a better way of doing this